modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -66,15 +72,21 @@ test.beforeEach(async ({apiHelpers}) => {
 			singleApproverWorkflowDefinition
 		);
 
-	workflowDefinitionId = workflowDefinition.id;
+	workflowDefinitionIds.push(workflowDefinition.id);
 });
 
 test.afterEach(async ({apiHelpers, scriptManagementPage}) => {
-	await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
-		workflowDefinitionId
-	);
+	for (const workflowDefinitionId of workflowDefinitionIds) {
+		await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
+			workflowDefinitionId
+		);
 
-	await scriptManagementPage.enableScriptManagementConfiguration();
+		workflowDefinitionIds = [];
```

Comment:
Hey @larissacribeiro, could you move it to just below the iteration?

Reply:
sure, thank you!!